src/api/bodhi_utils.js

```diff
@@ -0,0 +1,13 @@
+const { Utils } = require('qweb3');
+
+const BodhiUtils = {
```

Comment:
lets rename the file to `raw_transactions.js` and obj to `RawTransactions`. I originally placed the files/functions based on the qtumd help sections:

![image](https://user-images.githubusercontent.com/4350404/39595688-0b2532ae-4ec5-11e8-9312-9cba8f16fdcc.png)